integration/kubernetes/helm-generate.sh

```diff
@@ -199,10 +199,14 @@ function generateAllTemplates {
 
 workerFuse=false
 function main {
-  mode=$1
-  if [ "$3" = "--enable-workerFuse" ]; then
+  if [ "$3" = "--worker-fuse" ]; then
```

Comment:
@LuQQiu Since we only have one option, so directly compare $3 to "--worker-fuse" is fine. But if we have more options, we should not do this but use an array storing all options and use switch statement inside the loop. Do you think we should do this?